lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -0,0 +1,171 @@
+module Cerner
+    module Resources
+  
+    R4_RELATEDPERSON_ENTRY ||= {
+      "resourceType": "RelatedPerson",
+      "id": "3440007-4768010",
+      "meta": {
+          "versionId": 30,
+          "lastUpdated": "2019-01-09T15:25:32.000Z"
+      },
+      "text": {
+          "status": "generated",
+          "div": "<div><p><b>Related Person</b></p><p><b>Name</b>: CROWN, TEST A</p><p><b>DOB</b>: Feb 17, 1989</p><p><b>Gender</b>: male</p><p><b>Patient</b>: CROWN, ADAM</p></div>"
```

Comment:
need to use \&lt; and \&gt; instead of &lt; and &gt; otherwise the html gets rendered in the code container on the site

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/6d1dc18c06362fc770aea2991e8e317ec263a54e#diff-e77269d0d7c7c9767d78a8e4f8e121a0R13